examples/mysql/README.md

```diff
@@ -0,0 +1,77 @@
+# Overview
+
+OpenTelemetry HTTP Instrumentation allows the user to automatically collect trace data and export them to the backend of choice (we can use Zipkin or Jaeger for this example), to give observability to distributed systems.
```

Comment:
Looks like this is HTTP plugin README file, could you please change it for mysql?